Added Jimmithy to all maps

flufflycthu1u opened the pull request:
Moved the clouse on box and renamed it Jimmithy while adding it to all maps. Sprite courtesy of @metalgearman on discord and code courtesy of @SushiFish100. 

🆑 
add: Added a clouse named Jimmithy to the theatre on all maps.
/ 🆑

flufflycthu1u committed:
Added Jimmithy to all maps

Moved the clouse on box and renamed it Jimmithy while adding it to all maps.

AngryKnees commented:
Can we get this where Jimmithy is a child object of `/mob/living/simple_animal/hostile/retaliate/clown/russ/mouse`? Otherwise the badminbus won't be able to spawn Jimmithy.

flufflycthu1u committed:
Updated Jimmithy to be a subtype of clouse

For the badmemes.

Kmc2000 commented:
When the mouse dies, won't it leave behind a human clown corpse?

flufflycthu1u commented:
It won't drop a clown corpse because the loot table was overwritten in the clouse define. Discussed this on discord but I thought I'd just say it here for later documentation.

flufflycthu1u committed:
Merge remote-tracking branch 'upstream/master' into Clown-Pet

flufflycthu1u committed:
Added the actual Jimmithy for real this time

Added the latest version of Jimmithy and also updated its max health to match its initial health.
